proto/backend.proto

```diff
@@ -51,8 +51,9 @@ message DataQuery {
   string refId = 1;
   int64 maxDataPoints = 2;
   int64 intervalMS = 3;
-  TimeRange timeRange = 4; // NOTE: this moved from QueryRequest in v1
-  bytes json = 5; // Arbitrary structure (model?  modelJson in v1)
+  TimeRange timeRange = 4;
+  bytes json = 5;
+  string path = 6;
```

Comment:
strongly against using "path" here -- at least really unclear why "path" should be more special than any of the other common parameters used across many plugins ("type", "queryType", and "api" are also commonly used)

Reply:
I don't understand why "common" parameters should guide this object. (Also, to be clear, I currently understand "common" parameters as properties you have seen in used across various datasources / apps).

I think it makes more sense to think in terms of properties "reserved by grafana" and those that are not since we are making breaking changes to the query model anyways (moving time to every query).

If we can keep all custom properties at a nested level, the issue of name collision goes away and likely prevent future confusion and bugs.

If the idea is that we are really just replacing the way people use queryType, then maybe we should reserve that property.

Reply:
right -- given that the namespace is currently open to anything for plugins, if we are going to add a new "reserved by grafana" property, I am reluctant to pick a property that is already used widely (and differently) in different plugins